docs/user/reference/metrics/text.md

```diff
@@ -0,0 +1,150 @@
+# Text
+
+Records a single long Unicode text, used when the limits on `String` are too low.
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+## Important
+
+> This type should only be used in special cases. See [limitations](#limits) below.
+> It is best to talk to the Glean team before using this.
+
+## Recording API
+
+### `set`
+
+Sets a text metric to a specific value.
+
+{{#include ../../../shared/tab_header.md}}
+
+<div data-lang="Kotlin" class="tab"></div>
+
+<div data-lang="Java" class="tab"></div>
+
+<div data-lang="Swift" class="tab"></div>
+
+<div data-lang="Python" class="tab"></div>
+
+<div data-lang="JavaScript" class="tab">
+
+```js
+import * as display from "./path/to/generated/files/display.js";
```

Comment:
This example is a bit confusing as `display` is imported but the name of the metric used is `article`.